homeassistant/components/zha/light.py

```diff
@@ -175,7 +177,15 @@ def async_set_state(self, attr_id, attr_name, value):
             await self.async_accept_signal(
                 self._level_channel, SIGNAL_SET_LEVEL, self.set_level
             )
-        async_track_time_interval(self.hass, self.refresh, SCAN_INTERVAL)
+        refresh_interval = random.randint(*_REFRESH_INTERVAL)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5156575)